examples/train/dflash_qwen3_8b_sharegpt_online_10k_npu.sh

```
@@ -0,0 +1,99 @@
+
+# DFlash-specific parameters
+SPECULATOR_TYPE="dflash"
+BLOCK_SIZE=8
+MAX_ANCHORS=128
+NUM_LAYERS=5
+DRAFT_VOCAB_SIZE=32000
+TARGET_LAYER_IDS="2 18 33"  # Must match vLLM's eagle_aux_hidden_state_layer_ids
+
+# GPU assignments (online training needs separate GPUs for vLLM and training)
+VLLM_NPUS="0,1"
+TRAIN_GPUS="2,3,4,5"
```

Shellcheck (0.11.0)

[warning] TRAIN_GPUS appears unused. Verify use (or export if used externally).

(SC2034)